.github/workflows/pull_request.yml

```diff
@@ -5,16 +5,35 @@ name: Pull request tests.
 jobs:
   test:
     name: Pull request tests
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
     steps:
       - name: Checkout code
-        uses: actions/checkout@master
+        uses: actions/checkout@v3
+        with:
+          path: snapshot-streaming
+
+      - name: Checkout tessellation
+        uses: actions/checkout@v3
+        with:
+          path: tessellation
+          repository: Constellation-Labs/tessellation
+          ref: v1.9.1
```

Comment:
I think we need to scrape it from `project/Dependencies.scala` file, right?

Reply:
Well, I can change this and I thought about doing it right away, but I decided against it. My reasoning is that this is a _temporary_ solution and I think we shouldn't have any sophisticated temporary solutions. We should have permanent solutions - and that is publishing packages to GitHub packages repository.